stable/fluent-bit/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluent-bit
-version: 2.4.3
+version: 2.4.4
```

Comment:
Its a new feature, so should probably be 2.5.0. otherwise, looks good to me.

Reply:
Thanks @kfox1111 

Pushed a version update.